.github/workflows/audit.yaml

```diff
@@ -22,19 +22,21 @@ jobs:
       matrix:
         operating_system: ["ubuntu-latest", "macos-latest"]
     steps:
-    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # @v2
-    - uses: actions/setup-python@65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236 # @v2
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+    - uses: actions/setup-python@65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236
       with:
         python-version: ${{inputs.python_version}}
+        cache: 'pip'
```

Comment:
Note: this may or may not actually be helpful, see https://github.com/actions/setup-python/issues/330#issuecomment-1416883170